tests/src/system/basic/WskBasicTests.scala

```diff
@@ -466,6 +466,19 @@ class WskBasicTests
             wsk.trigger.list().stdout should include(name)
     }
 
+    it should "create a trigger using property file" in withAssetCleaner(wskprops) {
+        (wp, assetHelper) =>
+            val name = "listTriggers"
```

Comment:
Choose a more speaking name for the trigger to make logs easier to parse in case of test failures.